multiversx_sdk_network_providers/proxy_network_provider.py

```diff
@@ -86,21 +95,30 @@ def get_nonfungible_token_of_account(self, address: IAddress, collection: str, n
         return token
 
     def get_transaction(self, tx_hash: str) -> TransactionOnNetwork:
-        url = f"transaction/{tx_hash}?withResults=true"
-        response = self.do_get_generic(url).get('transaction', '')
-        transaction = TransactionOnNetwork.from_proxy_http_response(tx_hash, response)
+        data: Dict[str, Any] = {}
 
-        return transaction
+        def get_process_status():
```

Comment:
Can we refactor using `concurrent.futures`? That way, we can use `return_value = future.result()` instead of adding results to a shared dictionary. 

E.g. https://stackoverflow.com/a/58829816/1475331

Reply:
Now using `concurrent.futures`